ghpc-kenya-pediatric-hiv/js/line.js

```diff
@@ -0,0 +1,60 @@
+Highcharts.setOptions({
+  lang: {
+    thousandsSep: ','
+  }
+});
+
+Highcharts.chart('container', {
+
+    title: {
+        text: 'New Pediatric (Ages 0-14) HIV Infections in Kenya'
+    },
+    chart: {
+        style: {
+            fontFamily: "Source Sans Pro"
+        }
+    },
+    data: {
+        googleAPIKey: 'AIzaSyAImbihK2tiRewSFzuJTF_lcgPlGSr7zcg',
+        googleSpreadsheetKey: '1HA3hEmlb-rrp1_WsN5abxflDUbEhu_u4Zi1s3aaLaAg'
+    },
```

Comment:
Add the name of the tab that is used on the spreadsheet. Using the key that highcharts provide it ```googleSpreadsheetRange```. This way it won't break if in the future another tab is added to the spreadsheet.
```suggestion
googleSpreadsheetRange: "Sheet1"
},
```